.github/workflows/ci.yml

```
@@ -39,6 +39,16 @@ jobs:
 
       - run: bun install
 
+      - name: Generate bindings
+        run: |
+          mv src/binding.ts ${{ runner.temp }}/binding.ts
+          bun run generate
+          CMP=$(cmp src/binding.ts ${{ runner.temp }}/binding.ts) 
+          if [ -n "$CMP" ]; then
```

you could also just do

```suggestion
          if [[ $(git diff --stat src/binding.ts) != '' ]]; then
```